### PR TITLE
async-profiler 1.5 (new formula)

### PR DESCRIPTION
**Pull request by stringbean**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
CPU profiling and flame graph generator for Java.

### CODE REVIEW

File: Formula/async-profiler.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    (bin/"async-profiler").write <<~EOS
+      #!/bin/bash
+      "#{Formula["async-profiler"].libexec}/profiler.sh" "$@"
+    EOS
+    chmod 0555, bin/"async-profiler"
+  end
+
+  test do
+    system bin/"async-profiler", "-v"
```
Comment (fxcoudert) [inline Formula/async-profiler.rb]: We need a test that exercises the some of the functionality of the app.

File: Formula/async-profiler.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  sha256 "ffe8281f745a8e55bf9485c9ed9d30e5c4934174275cdb73b32e47cde35fffdd"
+
+  depends_on :java
+
+  def install
+    system "make"
+
+    libexec.install "build", "profiler.sh"
+
+    (bin/"async-profiler").write <<~EOS
```
Comment (fxcoudert) [inline Formula/async-profiler.rb]: Can we make a symlink instead?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/async-profiler.rb (hunk 1 above)
Comment (stringbean) [reply Formula/async-profiler.rb]: The only way we can properly exercise the app would be to start another Java process to attach to and get async-profiler to run a profile.

Are there any examples of tests for similar packages?

File: Formula/async-profiler.rb (hunk 2 above)
Comment (stringbean) [reply Formula/async-profiler.rb]: Changed.

**Pull request closed without merging**